tutorials/fileToJdbc/src/test/java/com/example/FileToJdbcApp/ApplicationTest.java

```diff
@@ -0,0 +1,139 @@
+  private static final String DB_DRIVER = "com.mysql.jdbc.Driver";
+  private static final String DB_URL = "jdbc:mysql://localhost:3306/testJdbc";
+  private static final String TABLE_NAME = "test_jdbc_table";
+  private static final String USER_NAME = "root";
+  private static final String PSW = "password";
+
+  @BeforeClass
+  public static void setup() {
+    try {
+      Class.forName(DB_DRIVER).newInstance();
+
+      Connection con = DriverManager.getConnection(DB_URL,USER_NAME,PSW);
+      Statement stmt = con.createStatement();
+
+      String createMetaTable = "CREATE TABLE IF NOT EXISTS " + JdbcTransactionalStore.DEFAULT_META_TABLE + " ( "
+              + JdbcTransactionalStore.DEFAULT_APP_ID_COL + " VARCHAR(100) NOT NULL, "
+              + JdbcTransactionalStore.DEFAULT_OPERATOR_ID_COL + " INT NOT NULL, "
+              + JdbcTransactionalStore.DEFAULT_WINDOW_COL + " BIGINT NOT NULL, "
+              + "UNIQUE (" + JdbcTransactionalStore.DEFAULT_APP_ID_COL + ", "
+              + JdbcTransactionalStore.DEFAULT_OPERATOR_ID_COL + ", " + JdbcTransactionalStore.DEFAULT_WINDOW_COL + ") "
+              + ")";
+      stmt.executeUpdate(createMetaTable);
+
+      String createTable = "CREATE TABLE IF NOT EXISTS " + TABLE_NAME
+              + "(ACCOUNT_NO INT(11) not NULL, NAME VARCHAR(255), AMOUNT INT(11))";
+      stmt.executeUpdate(createTable);
+
+    } catch (Throwable e) {
+      DTThrowable.rethrow(e);
+    }
+  }
+
+  public static void cleanTable()
+  {
+    try {
+      Connection con = DriverManager.getConnection(DB_URL,USER_NAME,PSW);
+      Statement stmt = con.createStatement();
+      String cleanTable = "truncate " + TABLE_NAME;
+      stmt.executeUpdate(cleanTable);
+    } catch (SQLException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public int getNumOfEventsInStore()
+  {
+    Connection con;
+    try {
+      con = DriverManager.getConnection(DB_URL,USER_NAME,PSW);
+      Statement stmt = con.createStatement();
+
+      String countQuery = "SELECT count(*) from " + TABLE_NAME;
+      ResultSet resultSet = stmt.executeQuery(countQuery);
+      resultSet.next();
+      return resultSet.getInt(1);
+    } catch (SQLException e) {
+      throw new RuntimeException("fetching count", e);
+    }
+  }
+
+  public void setConfig(Configuration conf)
+  {
+    conf.addResource(this.getClass().getResourceAsStream("/META-INF/properties.xml"));
+    conf.set("dt.operator.FileReader.prop.directory", INPUT_DIR);
+    conf.set("dt.application.FileToJdbcCsvParser.operator.CsvParser.port.out.attr.TUPLE_CLASS", TUPLE_CLASS);
+    conf.set("dt.application.FileToJdbcCustomParser.operator.CustomParser.prop.regexStr", REGEX_STR);
+    conf.set("dt.operator.JdbcOutput.port.input.attr.TUPLE_CLASS", TUPLE_CLASS);
+    conf.set("dt.operator.JdbcOutput.prop.store.databaseUrl", DB_URL);
+    conf.set("dt.operator.JdbcOutput.prop.tablename", TABLE_NAME);
+  }
+
+  @Test
+  public void testCsvParserApp() throws IOException, Exception {
```

Comment:
These tests fail for me since they expect MySql to be running:
```
mvn -Dtest=ApplicationTest#testCsvParserApp test
```

yields:
```
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:214)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:298)
```

Please look at how **jdbcIngest** does it (it uses DB_DRIVER = "org.hsqldb.jdbcDriver").
Here, I can run this test successfully without any DB server running:
```
mvn -Dtest=JdbcInputAppTest#testApplication test
```